test/unittests/commands/ingest_xmlupload/test_bulk_ingest_client.py

```diff
@@ -45,7 +45,6 @@ def test_upload_file_success(ingest_client: BulkIngestClient, requests_mock: Moc
     assert req.method == "POST"
     assert req.headers["Authorization"] == "Bearer mocked_token"
     assert req.headers["Content-Type"] == "application/octet-stream"
-    assert req.body == file_content.encode()
```

Comment:
IMHO there is no need to remove this assertion.

Reply:
Previously, the body contained all bytes of the entire file, which is now not the case anymore. I don't think there is an easy way to test the body, because now the file is streamed.